lucille-core/src/main/resources/log4j2.xml

```diff
@@ -2,17 +2,17 @@
 <Configuration>
     <Appenders>
         <Console name="console" target="SYSTEM_OUT">
-            <PatternLayout pattern="%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n" />
+            <PatternLayout pattern="%d{yy/MM/dd HH:mm:ss} %X{id} %p %c{1}: %m%n" />
```

Comment:
looks like we're putting the doc ID at the same place in the patternlayout where we put the Run ID in the LC-608 PR?  Suggest combining LC-608 and LC-609 into one unified PR so that we can be sure both sets of changes work together...